addons/website_form/static/src/snippets/s_website_form/options.js

```diff
@@ -547,15 +547,15 @@ options.registry.WebsiteFormEditor = FormEditor.extend({
      * @param {string} value
      * @param {string} fieldName
      */
-    addHiddenField: function (value, fieldName) {
-        this.$target.find(`.s_website_form_field.d-none:has(input[name="${fieldName}"])`).remove();
+    _addHiddenField: function (value, fieldName) {
+        this.$target.find(`.form-group:not(.s_website_form_field):has(input[name="${fieldName}"])`).remove();
```

Comment:
data-exclude is what I need !!!
I can keep it as it is and exclude .s_snippet_field.d-none
Or as you proposed replace d-none with the class but that would modify all templates like I am doing currently.
Considering 13.2 is not really stable it might be best to do the second option.

Reply:
As you wish